### ISSUE TITLE
Update bibliogram instances

### ISSUE THREAD

**Issue by B0pol** [benbusby/farside#63]
- Both https://insta.tromdienste.de/ and https://insta.trom.tf/ are blocked by a login page but are still shown
- some bibliogram instances on https://bibliogram.art/api/instances still work. E.g. bibliogram.art.

Bibliogram is [discontinued](https://cadence.moe/blog/2022-09-01-discontinuing-bibliogram) but

> What does discontinued mean?
> You can't look at profiles. You can still look at individual posts, but if this breaks in the future, I probably won't fix it.

In readme you say

> How It Works
> The app runs with an internally scheduled cron task that queries all instances for services defined in [services.json](https://github.com/benbusby/farside/blob/main/services.json) every 5 minutes. For each instance, as long as the instance takes <5 seconds to respond and returns a successful response code, the instance is added to a list of available instances for that particular service. If not, it is discarded until the next update period.

I could not find what the test is but I bet the test is querying a profile, that’s why all instances are down. Instead, you should use a post instead, they still work (for now)
https://bibliogram.art/p/Ch2WRmiLMjj/
